smooth/components/component_battery.py

```diff
@@ -37,34 +37,38 @@
 
 Maximum chargeable/dischargeable energy
 ---------------------------------------
-The maximum chargeable or dischargeable power [W] going in to or out of the battery is dependant
-on the C-rate and the capacity:
+The maximum power [W] going in to or out of the battery are dependant on
+the C-rate and the capacity:
 
 .. math::
-    P_{charge,max} = E_{cap} \\cdot C_{r,charge}
+    P_{in,max} = E_{cap} \\cdot C_{r,charge}
 
-    P_{discharge,max} = E_{cap} \\cdot C_{r,discharge}
-To ensure that the battery can be charged and discharged within the time frame inherently defined
-by the C-rate (1C : Full capacity can be (dis-)charged within one hour), the nominal value of
-the input-flow/output-flow between bus and battery needs to include the energy losses during the
-(dis-)charging process:
+    P_{out,max} = E_{cap} \\cdot C_{r,discharge}
+
+* :math:`P_{in,max}` = maximum power flowing from bus to battery [W]
+* :math:`E_{cap}` = battery capacity [Wh]
+* :math:`C_{r,charge}` = C-Rate for charging [W/Wh]
+* :math:`P_{out,max}` = maximum power flowing from battery to bus [W]
+* :math:`C_{r,discharge}` = C-Rate for discharging [W/Wh]
+
+.. figure:: /images/battery_losses.png
+    :width: 60 %
+    :alt: battery.png
+    :align: center
+
+    Fig.2: Diagram of the battery component including losses.
+
+The amount of energy, that the battery will be charged or discharged
+includes the energy losses during the (dis-)charging process:
```

Comment:
misleading formulation "includes" (possible: takes into account) or focus on in-flow and discharge:
```
Both, the amount of energy, that flows into the battery and the discharged energy,
include the energy losses during the (dis-)charging process, respectively:
```